cleverpush/src/main/java/com/cleverpush/banner/AppBannerModule.java

```diff
@@ -89,35 +89,39 @@ private AppBannerModule(Activity activity, String channel, boolean showDrafts) {
         this.sessions = this.getSessions();
 
         handlerThread.start();
-		handler = new Handler(handlerThread.getLooper());
+        handler = new Handler(handlerThread.getLooper());
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(CleverPush.context);
+        SharedPreferences.Editor editor = sharedPreferences.edit();
+        editor.putBoolean(CleverPushPreferences.APP_BANNERS_IS_POP_UP_SHOWING, false);
+        editor.commit();
     }
 
-	private void loadBanners() {
-    	loadBanners(null);
-	}
+    private void loadBanners() {
+        loadBanners(null);
```

Comment:
please only pass "" instead of null, because inside loadBanners you're both handling isEmpty() and null the same

Reply:
therefore there is no need to have double indicators for one behaviour